docker/Dockerfile.fireshape

```diff
@@ -8,6 +9,7 @@ RUN apt-get update \
     && apt-get -y install gmsh patchelf \
```

Comment:
We may add `gmsh` to the Firedrake base image if https://github.com/firedrakeproject/firedrake/pull/2732 lands (you don't need to change anything here, just FYI)